TopCP/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG BASE_IMAGE=gitlab-registry.cern.ch/atlas/athena/analysisbase:25.2.14
+FROM ${BASE_IMAGE} as base
+
+SHELL [ "/bin/bash", "-c" ]
+
+RUN yum update -y
+RUN yum install -y https://repo.opensciencegrid.org/osg/3.6/osg-3.6-el9-release-latest.rpm && yum install -y osg-ca-certs 
+RUN yum install -y nc
+
+COPY ./TopCPToolkit /TopCPToolkit
```

Comment:
We need to figure out how to build TopCPToolkit within the container. The only complicated part of this is that the relevant repository is password-protected, so we need to inject the access token into the build somehow. But I've verified that we can do the git checkout and build inside the analysisbase container.